launch/LMS1xx.launch

```diff
@@ -1,6 +1,13 @@
 <launch>
   <arg name="host" default="192.168.1.14" />
+  
+  <!-- start and end units are in degrees to keep with lms internal structure. They will be converted to rad in the node. -->
+  <arg name="start" default="-999.9" />
+  <arg name="end" default="999.9" />
```

Comment:
You are correct. I've been unable to figure out how to change the start/stop angles into the device directly. The one spot I found in the SOPAS software that has start angle as an input just rotated the frame. I've looked in the operating manual and wasn't able to find how to do it there either.